mypyc/ops_str.py

```diff
@@ -31,6 +32,27 @@
     error_kind=ERR_MAGIC,
     emit=simple_emit('{dest} = PyUnicode_Join({args[0]}, {args[1]});'))
 
+
+def emit_str_split(emitter: EmitterInterface, args: List[str], dest: str) -> None:
+    temp = emitter.temp_name()
+    emitter.emit_declaration('Py_ssize_t %s;' % temp)
+    emitter.emit_lines(
+        '%s = CPyTagged_AsSsize_t(%s);' % (temp, args[2]),
+        '%s = PyUnicode_Split(%s, %s, %s);' % (dest, args[0], args[1], temp))
```

Comment:
I think I'd prefer to define a helper function in `CPy.h` and then call that for this case. (The helper function could more easily handle the error case where the number is too big, also).